src/phoenix/experimental/evals/templates/template.py

```diff
@@ -45,49 +52,45 @@ def _get_delimiters(self, delimiters: List[str]) -> Tuple[str, str]:
         else:
             raise ValueError("delimiters must only contain 2 items in the list")
 
-    def _validate(self) -> None:
+    def _validate(self, text: str) -> None:
```

Comment:
suggestion: detach from the class to optimize for composition

Reply:
I'm with @axiomofjoy here, this method doesn't seem to be doing anything too useful per my understanding